src/main/scala/nl.knaw.dans.easy.authinfo/EasyAuthInfoServlet.scala

```diff
@@ -60,16 +61,16 @@ class EasyAuthInfoServlet(app: EasyAuthInfoApp) extends ScalatraServlet
   private def respond(uuid: UUID, path: Path, rights: Try[Option[CachedAuthInfo]]): ActionResult = {
     rights match {
       case Success(Some(CachedAuthInfo(json, Some(Failure(t))))) =>
-        logger.error(s"cache update failed for [$uuid/$path] reason: ${ t.getMessage.toOneLiner }")
+        logger.error(s"cache update failed for [$uuid/${path.escapePath}] reason: ${ t.getMessage.toOneLiner }")
         Ok(pretty(render(json)))
       case Success(Some(CachedAuthInfo(json, Some(Success(_))))) =>
-        logger.info(s"cache updated for [$uuid/$path]")
+        logger.info(s"cache updated for [$uuid/${path.escapePath}]")
         Ok(pretty(render(json)))
       case Success(Some(CachedAuthInfo(json, _))) =>
         Ok(pretty(render(json)))
-      case Success(None) => NotFound(s"$uuid/$path does not exist")
+      case Success(None) => NotFound(s"$uuid/${path.escapePath} does not exist")
       case Failure(HttpStatusException(message, HttpResponse(_, SERVICE_UNAVAILABLE_503, _))) => ServiceUnavailable(message)
-      case Failure(BagDoesNotExistException(uuid: UUID)) => NotFound(s"$uuid/$path does not exist")
+      case Failure(BagDoesNotExistException(uuid: UUID)) => NotFound(s"$uuid/${path.escapePath} does not exist")
```

Comment:
For a not-found error message I would like a not escaped path to simplify trouble shooting. Did not yet form an opinion about the other logging. Think what it tries to tell: _what_ is indexed or _how_ is it indexed?

Reply:
@jo-pol : the `/` should **definetely not** be escaped. The idea of escapePath is to escape the components op the path.

Reply:
@jo-pol: the string with the escaped path is the identifier for the thing being indexed.  `<uuid>/<non-escaped-path>` is nothing, so it would promote the misunderstanding about the item-ids which is apparently still present. I would be OK with debug-level log-message that says something like: "Item `<idem-id here>` not found. This points to bag: `<uuid>` file: `<unescaped file-path>`.